web-api/openapi/front_office.yaml

```
@@ -48,32 +48,83 @@ tags:
   - name: front office
     description: Handles interactions between users and documents and allows to fetch personalized documents per user.
     x-traitTag: true
+  - name: recommendation
+    x-displayName: Recommendation
   - name: interaction
     x-displayName: Interaction
   - name: search
     x-displayName: Search
 x-tagGroups:
-  - name: Interaction
+  - name: Recommendation API
     tags:
+      - recommendation
       - interaction
-  - name: Search
+  - name: Search API
     tags:
       - search
 
 security:
   - ApiKeyAuth: []
 
 paths:
+  /users/{user_id}/recommendations:
+    parameters:
+      - $ref: './parameters/path/id.yml#/UserId'
+
+    post:
+      tags:
+        - front office
+        - recommendation
+      summary: Provides recommendations for a given user
+      description: |-
+        Finds a number of recommendations for the given `user_id`.
+
+        Recommendations are based on snippets and each recommendation contains the snippet id it is based on as
+        well as a score. A higher score means that the document matches the preferences of the user better.
+        Scores can be compared only with other scores that belong to the same request; comparing scores of documents that have been obtained through different requests can lead to unexpected results.
+
+        Depending on the request parameters the recommendation can also include additional fields.
+
+        The recommendations also contain the properties of the snippets parent document if this is requested and the properties are not empty.
+
+        Documents that have been interacted with by the user are filtered out from the result.
+
+        Note that you can request personalized documents for a specific `user_id`, only after that same `user_id` has made enough interactions via our system.
+      operationId: getRecommendations
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RecommendationRequest'
+      responses:
+        '200':
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RecommendationResponse'
+        '400':
+          $ref: './responses/generic.yml#/BadRequest'
+        '409':
+          description: Impossible to create a personalized documents for the user.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RecommendationError'
+
   /users/{user_id}/personalized_documents:
     parameters:
       - $ref: './parameters/path/id.yml#/UserId'
 
     post:
       tags:
         - front office
-        - search
+        - recommendation
+      deprecated: true
       summary: Personalize documents for the user
       description: |-
+        Use `/users/{user_id}/recommendations` instead.
```

```suggestion
        This endpoint has been renamed as `/users/{user_id}/recommendations`, use it instead.
```
I would delete everything else in the description